recipes/qudida/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please switch to the GitHub source release url. The LICENSE is included in the GitHub source, so it will be automatically updated and you won't manually include it in the recipe.